### ISSUE TITLE
Keras Model.evaluate progress bar doesn't work in graph mode in TF2.2

### ISSUE THREAD

**Issue by drasmuss** [tensorflow/tensorflow#39456]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.2.0
- Python version: 3.7
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**

Calling `Model.evaluate` in graph mode doesn't produce any stdout.

**Describe the expected behavior**

A progress bar should be displayed along with the loss values, as in TF<2.2.

**Standalone code to reproduce the issue**

https://colab.research.google.com/gist/drasmuss/b4d074f13c4bed294a1403f351e4d263

**Comment by Saduf2019** [comment]
@drasmuss 
Please move this issue to closed status as associated pr is in merged status.

**Comment by drasmuss** [comment]
Which PR is that?

**Comment by Saduf2019** [comment]
#153 above shows that it is merged.

**Comment by drasmuss** [comment]
https://github.com/tensorflow/tensorflow/issues/153 is an issue, not a PR, and seems unrelated to this issue. I think you might be referring to https://github.com/nengo/nengo-dl/pull/153, which is a PR in a completely different project?

**Comment by nikitamaia** [comment]
Hi @drasmuss, disabling eager execution is generally not encouraged. Is there a specific reason for your use case that you need `tf.compat.v1.disable_eager_execution()`?

**Comment by drasmuss** [comment]
It is for performance reasons, disable_eager_execution results in faster performance. See discussion and reproducible example here https://github.com/tensorflow/tensorflow/issues/44563#issuecomment-722635251

**Comment by saikumarchalla** [comment]
Was able to reproduce the issue using TF version 2.5 and Nightly as well. Please find the gist [here](https://colab.research.google.com/gist/saikumarchalla/cd645f5b0519cceccb709c56e1d6f2f3/untitled79.ipynb).

**Comment by tensorflowbutler** [comment]
Hi There,

 This is a stale issue. As you are using an older version of tensorflow, we are checking to see if you still need help on this issue. Please test the issue with the latest TensorFlow (TF2.7 and tf-nightly). If the issue still persists with the newer versions of TF, please feel free to open it in [keras-team/keras](https://github.com/keras-team/keras/issues) repository by providing details about the issue and a standalone code to reproduce the issue. Thanks! 

 Please note that Keras development has moved to a separate Keras-team/keras repository to focus entirely on only Keras. Thanks!

**Comment by drasmuss** [comment]
The issue still persists in TF 2.7, I created a new issue in the Keras repo here [keras-team/keras#15792](https://github.com/keras-team/keras/issues/15793).
